frontend/admin/src/js/components/user/GeneralInfoTabDialogs.tsx

```diff
@@ -0,0 +1,690 @@
+            description:
+              "Toast for successful add user to pool on view-user page",
+          }),
+        );
+        resetAndClose();
+      })
+      .catch(() => {
+        toast.error(
+          intl.formatMessage({
+            defaultMessage: "Failed adding user",
+            description: "Toast for failed add user to pool on view-user page",
+          }),
+        );
+      });
+  };
+
+  return (
+    <Dialog
+      title={intl.formatMessage({
+        defaultMessage: "Add to different pool",
+        description: "title for add to pool dialog on view-user page",
+      })}
+      color="ts-primary"
+      isOpen={isVisible}
+      onDismiss={resetAndClose}
+      footer={
+        <div
+          data-h2-display="b(flex)"
+          data-h2-justify-content="b(space-between))"
+        >
+          <CloseDialogButton close={resetAndClose} />
+          <ConfirmDialogButton
+            onConfirm={handleSubmit}
+            title={intl.formatMessage({
+              defaultMessage: "Add to new pool",
+              description: "Confirmation button for add to pool dialog",
+            })}
+          />
+        </div>
+      }
+    >
+      <p>
+        {intl.formatMessage({
+          defaultMessage: "You're about to add this user to a different pool:",
+          description: "First section of text on the add user to pool dialog",
+        })}
+      </p>
+      <p>
+        - {user.firstName} {user.lastName}
+      </p>
+      <p>
+        {intl.formatMessage({
+          defaultMessage: "Choose pool:",
+          description: "Second section of text on the add user to pool dialog",
+        })}
+      </p>
+      <div data-h2-margin="b(bottom, xxs)">
+        <InputWrapper
+          inputId="pool"
+          label={intl.formatMessage({
+            defaultMessage: "Pools",
+            description:
+              "Label displayed on the pools field of the add user to pool dialog",
+          })}
+          required
+        >
+          <select
+            data-h2-radius="b(s)"
+            data-h2-padding="b(all, xs)"
+            data-h2-font-size="b(normal)"
+            data-h2-width="b(100)"
+            id="pool"
+            defaultValue=""
+            onChange={(e) => setSelectedPool(e.target.value)}
+          >
+            <option value="" disabled>
+              {intl.formatMessage({
+                defaultMessage: "Select a pool...",
+                description:
+                  "Placeholder displayed on the pool field of the add user to pool dialog.",
+              })}
+            </option>
+            {pools.map(
+              (pool) =>
+                !isEmpty(pool?.name?.[locale]) && (
+                  <option
+                    data-h2-font-family="b(sans)"
+                    key={pool?.id}
+                    value={pool?.id}
+                  >
+                    {pool?.name?.[locale]}
+                  </option>
+                ),
+            )}
```

Comment:
Since we can't add a user to a pool they are already in, should we filter those out of the available options, or disable them?

Reply:
Yeah, it's on my list.